src/test/java/teammates/test/cases/action/AdminInstructorAccountAddActionTest.java

```diff
@@ -178,8 +178,9 @@ private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId
                                      a, new Object[] { instructorEmailOrProposedCourseId, maximumIdLength });
     }
 
-    private AdminInstructorAccountAddAction getAction(String... parameters) {
-        return (AdminInstructorAccountAddAction) gaeSimulation.getActionObject(uri, parameters);
+    @Override
+    protected AdminInstructorAccountAddAction getAction(String... parameters) {
```

Comment:
Let me just do it here, at least for this method `getAction`. All instances of this method are touched in this PR anyway.